.github/workflows/Manager-CI.yaml

```diff
@@ -158,6 +158,9 @@ jobs:
           SLACK_CHANNEL: ${{ secrets.SLACK_CHANNEL }}
       - name: Run integration tests
         env:
+          ANSIBLE_ENDPOINT: ${{ secrets.ANSIBLE_ENDPOINT }}
+          ANSIBLE_USERNAME: ${{ secrets.ANSIBLE_USERNAME }}
+          ANSIBLE_PASSWORD: ${{ secrets.ANSIBLE_PASSWORD }}
```

Comment:
Do we have Ansible available for upstream testing?

Reply:
@jiripetrlik What about for local dev? Does the team need to know about the update for these tests?

Reply:
@robpblake Values for these environment variables were added to our "OpenBridge - Environment Details" spreadsheet.